From 25389a328c0232dd74f147635f9ac11f9f31a0b1 Mon Sep 17 00:00:00 2001
From: Stephen Chen <stephen@radxa.com>
Date: Wed, 27 Sep 2023 16:18:51 +0800
Subject: [PATCH] arm64: dts: rockchip: add chosen node for radxa boards

Signed-off-by: Stephen Chen <stephen@radxa.com>

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dtsi
index 75fea93f4f23..9a7e7c1616cd 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dtsi
@@ -11,6 +11,10 @@
 #include "rk3399-linux.dtsi"
 
 / {
+	chosen {
+		bootargs = "earlycon=uart8250,mmio32,0xff1a0000 console=ttyFIQ0 rw root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait coherent_pool=1m";
+	};
+
 	clkin_gmac: external-gmac-clock {
 		compatible = "fixed-clock";
 		clock-frequency = <125000000>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-v11.dts b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-v11.dts
index 912c175ad02b..1e7071f229b8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-v11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-v11.dts
@@ -22,8 +22,6 @@ / {
 	model = "Radxa ROCK 5B";
 	compatible = "radxa,rock-5b", "rockchip,rk3588";
 
-	/delete-node/ chosen;
-
 	vcc12v_dcin: vcc12v-dcin {
 		compatible = "regulator-fixed";
 		regulator-name = "vcc12v_dcin";
diff --git a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
index 6d96239e172e..5cb1b8fe0088 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
@@ -24,8 +24,6 @@ / {
 	model = "Radxa ROCK 5B";
 	compatible = "radxa,rock-5b", "rockchip,rk3588";
 
-	/delete-node/ chosen;
-
 	fan0: pwm-fan {
 		compatible = "pwm-fan";
 		#cooling-cells = <2>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-io.dts b/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-io.dts
index 188bd7b9baf5..40b370ac7a9b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-io.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-io.dts
@@ -13,8 +13,6 @@ / {
 	model = "Radxa CM5 IO";
 	compatible = "radxa,cm5-io", "radxa,cm5", "rockchip,rk3588";
 
-	/delete-node/ chosen;
-
 	vcc12v_dcin: vcc12v-dcin {
 		compatible = "regulator-fixed";
 		regulator-name = "vcc12v_dcin";
diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-rpi-cm4-io.dts b/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-rpi-cm4-io.dts
index 50b74f012c04..24bcbc136a70 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-rpi-cm4-io.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-radxa-cm5-rpi-cm4-io.dts
@@ -13,8 +13,6 @@ / {
 	model = "Radxa CM5 RPI CM4 IO";
 	compatible = "radxa,cm5-rpi-cm4-io", "rockchip,rk3588";
 
-	/delete-node/ chosen;
-
 	vcc12v_dcin: vcc12v-dcin {
 		compatible = "regulator-fixed";
 		regulator-name = "vcc12v_dcin";
diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-radxa-nx5-module.dtsi b/arch/arm64/boot/dts/rockchip/rk3588s-radxa-nx5-module.dtsi
index f913e75e3c07..4bec0e48a364 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588s-radxa-nx5-module.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-radxa-nx5-module.dtsi
@@ -28,8 +28,6 @@ aliases {
 		mmc2 = &sdio;
 	};
 
-	/delete-node/ chosen;
-
 	vcc12v_dcin: vcc12v-dcin {
 		compatible = "regulator-fixed";
 		regulator-name = "vcc12v_dcin";
@@ -406,4 +404,4 @@ rtc_int: rtc-int {
 			rockchip,pins = <0 RK_PB0 RK_FUNC_GPIO &pcfg_pull_none>;
 		};
 	};
-};
\ No newline at end of file
+};
diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-rock-5a.dts b/arch/arm64/boot/dts/rockchip/rk3588s-rock-5a.dts
index 5de05bda2caa..55b6092c9bd0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588s-rock-5a.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-rock-5a.dts
@@ -23,8 +23,6 @@ / {
 	model = "Radxa ROCK 5A";
 	compatible = "radxa,rock-5a", "rockchip,rk3588";
 
-	/delete-node/ chosen;
-
 	fan0: pwm-fan {
 		compatible = "pwm-fan";
 		#cooling-cells = <2>;
-- 
2.25.1

