From 9dd2bc93cac35fc9268294700f291991c7e85183 Mon Sep 17 00:00:00 2001
From: Stephen Chen <stephen@radxa.com>
Date: Mon, 3 Jul 2023 20:24:24 +0800
Subject: [PATCH] common: fix build error

Signed-off-by: Stephen Chen <stephen@radxa.com>

diff --git a/common/lrz.c b/common/lrz.c
index 1c175f9c7b9c..a32d60d68a26 100644
--- a/common/lrz.c
+++ b/common/lrz.c
@@ -2398,7 +2398,7 @@ moredata:
 					|| zi->bytes_received > last_bps / 2 + last_rxbytes)) {
 				int minleft =  0;
 				int secleft =  0;
-				time_t now;
+				time_t now = 0;
 				ulong d;
 				//xil_printf("%s(): L%d debug\n\r", __func__, __LINE__);
 				d=timing(0,&now);
-- 
2.25.1

