From 0799e4347d1117e9ad490ff564c61aff95c55588 Mon Sep 17 00:00:00 2001
From: Stephen Chen <stephen@radxa.com>
Date: Mon, 3 Jul 2023 20:30:16 +0800
Subject: [PATCH] common: rbsb: fix build error

Signed-off-by: Stephen Chen <stephen@radxa.com>

diff --git a/common/rbsb.c b/common/rbsb.c
index b80201405307..32ed48370606 100644
--- a/common/rbsb.c
+++ b/common/rbsb.c
@@ -480,9 +480,9 @@ io_mode(int fd, int n)
 #ifdef TIOCFLUSH
 		{ int x=1; ioctl(fd,TIOCFLUSH,&x); }
 #endif
+		return OK;
 #endif
 
-		return OK;
 	default:
 		return ERROR;
 	}
-- 
2.25.1

